TheKeyOAuth2/TheKeyOAuth2Client.m

```diff
@@ -14,15 +14,15 @@
 
 /* TheKey OAuth2 Settings */
 static NSString *const TheKeyOAuth2ServiceProvider = @"TheKey";
-static NSString *const TheKeyOAuth2RedirectURI     = @"thekey:/oauth/mobile/ios";
-static NSString *const TheKeyOAuth2Scope           = @"fullticket";
+static NSString *const TheKeyOAuth2RedirectURI     = @"thekey:/oauth/client/public";
```

Comment:
this redirectUri is still wrong. previously it had been a hardcoded fixed value, now it should be dynamic based on the configured CAS server url